modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize and configure an existing project access roles name by editing the YAML code, and add an `console.openshift.io/display-name: Display name` annotation to the cluster.
```

Comment:
To sum up all my confusions: the current introduction includes a tip about displaying access roles names, it doesn't describe what the procedure is about.

Also, customize and configure are basically the same here, are they not?

```suggestion
You can customize cluster roles of existing projects by editing the YAML code.
```